### ISSUE TITLE
Double-clicking on job in Jobs list crashed Plotter

### ISSUE THREAD

**Issue by Jaga-Telesin** [MrPig91/PSChiaPlotter#66]
Was just investigating my ability to remove/cancel a job.  Double-clicking the first (of two) jobs under the "Jobs" heading caused a fatal and immediate crash of the plotter.  Unfortunately I didn't get a capture of the error screen produced.  I quickly re-started the plotter (the Powershell instance was still running), only to find the PSChiaPlotter interface devoid of any running job information.

I allowed it to run for a couple minutes, but no log info was populated and no jobs showed up.  The chia.exe job processes are still running fortunately, so only temporary plotting time was lost.  But I have no idea now which PID belongs to which job, so need to wait for all to complete before restarting jobs with the Plotter.

I checked the plotter log directory, but there is no crash dump present to retrieve info from.  The Event Viewer for Windows Powershell only lists Info entries, and has no record of the crash at all.

**Comment by MrPig91** [comment]
I will check this out when I get home. So you double clicked the job in the datagrid? I do not believe there is any double click actions assigned to that, but I will try to reproduce the issue and find out what caused it. Thank you for reporting this. I will say that the logging on this applications is very lack luster at the moment, but I do plan to add better logging n the future.

**Comment by Jaga-Telesin** [comment]
Cool, looking forward to hearing what you find out.  I'd try it again here, but I'd rather have the Plots finish first than risk somehow dumping them.  :)

And yes, simply double-clicking on a job in the top area of the data grid is what produced the crash.  Note:  this is on my W10 machine which is solid as a rock, so I highly doubt it was just a process unexpectedly terminating.

For reference - I was investigating my ability to remove a job (and any queued plot tasks) while leaving another job and it's tasks running.  I have 4 HDD and 1 SSD I plot to, and I try to keep wear down on the SSD.  Being able to tell the SSD's job "terminate when the current plot is finished transferring" would be handy, though not critical.

**Comment by rathishsiva** [comment]
Yes, I faced the same issue when I double clicked. It just crashed and although the process was still running, when I relaunched the plotter, the jobs were not visible in the plotter. I was trying to explore the app in the hope that I could remove an individual job.

**Comment by MrPig91** [comment]
The double clicking issue "should" be fixed in the latest release. I removed some old code that I was going to use for hyperlinks for the temp and final directory paths, but I think this was the thing that was causing issue. I have also made the datagrids's readonly since double clicking on the job datagrid would create a new job grid row. I do plan to add the feature of being able to remove jobs and queues in the future.
